lib/tweetsclient/postgres_track.py

```diff
@@ -0,0 +1,56 @@
+import psycopg2
+import logbook
+
+import tweetsclient
+
+log = logbook.Logger(__name__)
+
+class PostgresTrackPlugin(tweetsclient.TrackPlugin):
+    def _get_database(self):
+        log.debug('Connecting to database')
+        conn = psycopg2.connect(
+            dbname=self.config.get('database', 'database'),
+            user=self.config.get('database', 'username'),
+            password=self.config.get('database', 'password'),
+            host=self.config.get('database', 'host'),
+            port=self.config.get('database', 'port')
+        )
+        return conn
+    
+    def _query(self, connection, table_name, field_name, conditions = None):
+        cursor = connection.cursor()
+        q = "SELECT %s FROM %s" % (field_name, table_name)
+        if conditions:
+            q = q + " WHERE %s" % (conditions)
+        log.debug("Executing query: {0}", q)
+        cursor.execute(q)
+        return [str(t[0]) for t in cursor.fetchall()]
+    
+    def _get_trackings(self):
+        tbl = self.config.get('database', 'table')
+        fld = self.config.get('database', 'field')
+        cnd = self.config.get('database', 'conditions')
+        conn = self._get_database()
+        return self._query(conn, tbl, fld, cnd)
+    
+    def get_accounts(self):                    
+        tbl = self.config.get('database', 'table')                                                 
+        fld = self.config.get('database', 'field-fetch')                                                 
+        cnd = self.config.get('database', 'conditions-fetch')                                            
+        conn = self._get_database()
+        cursor = conn.cursor()
+        q = "SELECT %s FROM %s WHERE %s ORDER BY id DESC" % (fld, tbl, cnd)
```

Comment:
I know `fld`, `tbl`, etc., come from config BUT I'd be happy if [we don't format SQL statements in Python like this](https://www.psycopg.org/docs/usage.html#the-problem-with-the-query-parameters)... We should either let cursor.execute handle the formatting or use [sql](https://www.psycopg.org/docs/sql.html) module for composition.